miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java

```diff
@@ -506,4 +527,160 @@ public void setRunDao(RunStore runDao) {
     this.runDao = runDao;
   }
 
+  @Override
+  public boolean processNotification(NotificationDto dto) throws IOException, MisoNamingException {
+    final Date now = new Date();
+    User user = securityManager.getUserByLoginName("notification");
+    SecurityContextHolder.getContext().setAuthentication(new SuperuserAuthentication(user));
+    final Run run;
+
+    Run runFromDb = runDao.getByAlias(dto.getRunName());
```

Comment:
I'm concerned about the ambiguity of the DTO field being called `runName` but using it to represent the run's `alias`. Is it possible to disambiguate this, maybe by renaming the DTO field to `runAlias`?